client/card-readers/preview-receipt/preview-receipt.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { useState, useEffect } from '@wordpress/element';
+import { Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+/**
+ * Internal dependencies
+ */
+import { LoadableBlock } from '../../components/loadable';
+import PrintedReceiptPreviewer from 'wcpay/card-readers/preview-receipt/printed-receipt-previewer';
+import {
+	useAccountBusinessSupportAddress,
+	useAccountBusinessName,
+	useAccountBusinessURL,
+	useAccountBusinessSupportEmail,
+	useAccountBusinessSupportPhone,
+} from '../../data';
+import { FetchReceiptPayload } from 'wcpay/types/card-readers';
+
+async function fetchReceiptHtml(
+	payload: FetchReceiptPayload
+): Promise< string > {
+	const path = '/wc/v3/payments/readers/receipts/print/preview';
+	return apiFetch( { path, data: payload, method: 'post' } );
+}
+
+const PreviewReceipt = (): JSX.Element => {
+	const [ receiptHtml, setReceiptHtml ] = useState< string >( '' );
+	const [ isLoading, setIsLoading ] = useState( true );
```

Comment:
Is it possible to make the typing here consistent with the above statement?

Reply:
Done thanks!